src/helper/window.js

```diff
@@ -10,8 +10,19 @@ function getWindow() {
   return global.window;
 }
 
+function getOrigin() {
+  var location = global.window.location;
+  var origin = location.origin;
+  if (!origin) {
+    origin =
+      location.protocol + '//' + location.hostname + (location.port ? ':' + location.port : '');
```

Comment:
Why the new line?

Reply:
[prettier](https://github.com/prettier/prettier) auto formatting.